README.md

```diff
@@ -1,119 +1,189 @@
+| Danish Fungi 2020        | [LINK [~110GB]](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20-train_val.tar.gz) | [LINK [~6.5GB]](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20-train_val.tar.gz) | [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20-metadata.zip) |
+| Danish Fungi 2020 – Mini | [LINK [~12.5GB]](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20M-images.tar.gz)  | ---                                                                                     | [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20M-metadata.zip) |
+| Danish Fungi 2024        | ❗Same as for DF20❗                                                                        | ❗Same as for DF20❗                                                                      | [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DanishFungi2024.zip)                                                           |
+| Danish Fungi 2024 – Mini | ❗Same as for DF20 – Mini❗                                                                 | ---                                                                                     |    [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DanishFungi2024-Mini.zip)                                                   |
+
+
+To download the dataset files in CMD, use:
+```
+wget --no-check-certificate URL
+```
 
-Available at -> https://sites.google.com/view/danish-fungi-dataset
+In order to support research in fine-grained plant classification and to allow full reproducibility of our results, we share the training scripts and data tools.
+- Checkpoints are available at [Hugging Face Hub Repository](https://huggingface.co/BVRA).
+- Train and Validation logs are available at [Weights & Biases Workspace](https://wandb.ai/zcu_cv/DanishFungi2023).
 
-## Training
 
-1. Download PyTorch NGC Docker Image and RUN docker container
 
+## Installation
+Python 3.10+ is required.
+### Local instalation
+1. Install dependencies
 ```
-docker pull nvcr.io/nvidia/pytorch:21.07-py3
-docker run --gpus all -it --rm -v local_dir:container_dir nvcr.io/nvidia/pytorch:21.07-py3
+pip install -r requirements.txt
 ```
-
-2. Install dependencies inside docker container
-
+2. Login to [Weights & Biases](https://wandb.ai/site) to log results [*optional].
 ```
-pip install pandas seaborn timm albumentation tqdm efficientnet_pytorch pretrainedmodels
+import wandb
+wandb.login()
 ```
-3. RUN jupyterlab and start training / experiments
+3. Login to [Hugging Face Hub](https://huggingface.co/) to save and download model checkpoints [*optional].
 ```
-jupyter lab --ip 0.0.0.0 --port 8888 --allow-root
+import huggingface_hub
+huggingface_hub.login()
 ```
-* Check your paths! 
 
-## Results
+## Training
+For training navigate to `./training` folder.
+To run the training you can use the provided `train.ipynb` notebook or `train.py` CLI.
+In both you have to:
+* Specify valid paths, wandb settings, etc. in **train.ipynb** and run.
```

Comment:
maybe there could be an example like
`python train.py .....`

Reply:
can we be more specific about what exactly they need to set?

Reply:
<img width="490" alt="image" src="https://github.com/user-attachments/assets/975d1830-9a92-43be-9546-d6d40ed6d9f5">
I guess these have names based on how users download it, maybe we can keep the filenames in there and let users specify the path only?

Reply:
Ok the python train.py exasmple is in the notebook, I would just mention it here maybe